templates/kdump.j2

```diff
@@ -12,6 +12,8 @@ KDUMP_COMMANDLINE_APPEND="irqpoll nr_cpus=1 reset_devices cgroup_disable=memory
 
 MKDUMPRD_ARGS=""
 
+{% elif ansible_os_family == 'RedHat' and ansible_distribution_version|int < 7 %}
```

Comment:
aren't you just duplicating the `{%elif ` line above?

Reply:
Not at all, the `elif` above is for `ansible_distribution == 'RedHat'`, which is RHEL. And this `elif` is for `ansible_os_family == 'RedHat'`, which is CentOS, RHEL, and Fedora. So in this case, `KDUMP_BOOTDIR` will go to both CentOS and RHEL 6.

Reply:
I missed this subtle difference. Still, are you sure this is right? First, `ansible_distribution == 'RedHat'` does cover only RHEL, but shouldn't it cover also CentOS, as CentOS should be in every respect identical to RHEL? Second, on RHEL, the  `ansible_distribution == 'RedHat'` branch will be taken, but it will prevent the `ansible_os_family == 'RedHat'` from being taken and `KDUMP_BOOTDIR="/boot"` will not get into the file on RHEL 6, which is a regression.

Reply:
I think you should do just s/distribution/os_family/ in the first condition instead of adding the second condition.

Reply:
I've done this in the first place because was thinking that the condition matched RHEL 6 but not CentOS 6 intentionally. I fixed this now, but it didn't fix the issue with tests_ssh.yml. I have no idea how to fix this. `/sbin/mkdumprd` for some reason searches for `ifcfg-dev`.

Reply:
No, I think the original author has not considered that we need to support CentOS as well and that it was not matched by this condition.